website/docs/reference/widgets/code-scanner.md

```diff
@@ -27,14 +28,22 @@ We support the following formats for QR and barcodes:
-| [Disabled](./README.md#disabled)                                        | Binding & Formatting | Disables input to the widget. The widget will remain visible to the user but a user input won't be allowed. Learn more about [Disable Property](./README.md#disabled)       | `{{CodeScanner.isDisabled}}` |
-| [Animate Loading](./README.md) | Formatting  | Controls the loading of the widget.  | NA   |
-| [Tooltip](./README.md#tooltip) | Formatting  | It sets a tooltip for the widget. You can add hints or extra information about the required input from the user. Learn more about [Tooltip](./README.md#tooltip) | NA  |
-| [Value](code-scanner.md#value)  | Binding   | Fetches the value of scanned code   | `{{CodeScanner.value}}`  |
+| Property                                         | Type                 | Description                                                                                                                                                               | Code Snippet                 |
+|--------------------------------------------------|----------------------|---------------------------------------------------------------------------------------------------------------------------------------------------------------------------|------------------------------|
+| [Scanner Layout](code-scanner.md#scanner-layout) | Formatting           | Sets how the code scanner will look and behave.                                                                                                                           | NA                           |
+| [Text](code-scanner.md#text)                     | Formatting           | Sets the label text of the widget.                                                                                                                                        | NA                           |
+| [Visible](./README.md#visible)                   | Binding & Formatting | Controls widget's visibility on the page. When turned off, the widget won't be visible when the app is published. Learn more about [Visible Property](./README.md#visible | `{{CodeScanner.isVisible}}`  |
+| [Disabled](./README.md#disabled)                 | Binding & Formatting | Disables input to the widget. The widget will remain visible to the user but a user input won't be allowed. Learn more about [Disable Property](./README.md#disabled)     | `{{CodeScanner.isDisabled}}` |
+| [Animate Loading](./README.md)                   | Formatting           | Controls the loading of the widget.                                                                                                                                       | NA                           |
+| [Tooltip](./README.md#tooltip)                   | Formatting           | It sets a tooltip for the widget. You can add hints or extra information about the required input from the user. Learn more about [Tooltip](./README.md#tooltip)          | NA                           |
```

Comment:
Where does it show up on the widget?

Reply:
Tooltip is visible only when Scanner Layout is set to "Click to Scan"

<img width="616" alt="CleanShot 2022-12-08 at 14 52 13@2x" src="https://user-images.githubusercontent.com/22471214/206408146-6300a61d-0373-4074-85c1-3c67fb7e73e9.png">

Moreover, we hide the tooltip property in the property pan when the scanner layout is always on.

<img width="272" alt="image" src="https://user-images.githubusercontent.com/22471214/206408539-7239e0b6-6539-47db-a7f4-d1c082e48e66.png">